<ui version="4.0" >
 <class>IDlgConfig</class>
 <widget class="QDialog" name="IDlgConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Environment</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Font</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="labelFont" >
            <property name="text" >
             <string>ABCDEFGHabcdefgh123456789</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolFont" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Units</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QRadioButton" name="radioMetric" >
            <property name="text" >
             <string>Metric</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioNautic" >
            <property name="text" >
             <string>Nautic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioImperial" >
            <property name="text" >
             <string>Imperial</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Mouse Wheel</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QCheckBox" name="checkFlipMouseWheel" >
            <property name="text" >
             <string>Flip</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Proxy</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="checkProxy" >
            <property name="text" >
             <string>enable</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineProxyURL" />
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineProxyPort" >
              <property name="inputMask" >
               <string>00000; </string>
              </property>
              <property name="text" >
               <string>8080</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Browser</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QComboBox" name="comboBrowser" >
            <item>
             <property name="text" >
              <string>Firefox</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Konqueror</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>other (needs command line)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineBrowserCmd" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>my command %s</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>%s will be replaced by URL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <attribute name="title" >
       <string>Device &amp;&amp; Xfer</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QComboBox" name="comboDevice" />
       </item>
       <item>
        <widget class="QGroupBox" name="groupDevice" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Device</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelDevIPAddr" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>IP Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLineEdit" name="lineDevIPAddr" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelDevIPPort" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineDevIPPort" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>4242</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelDevSerialPort" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Serial Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineDevSerialPort" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="labelDevType" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QComboBox" name="comboDevType" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolPlugins" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1" >
           <widget class="QLabel" name="labelMessage" >
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignJustify|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Upload All</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QCheckBox" name="checkUploadWpt" >
            <property name="text" >
             <string>Waypoints</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkUploadTrk" >
            <property name="text" >
             <string>Tracks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkUploadRte" >
            <property name="text" >
             <string>Routes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Download All</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QCheckBox" name="checkDownloadWpt" >
            <property name="text" >
             <string>Waypoints</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkDownloadTrk" >
            <property name="text" >
             <string>Tracks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkDownloadRte" >
            <property name="text" >
             <string>Routes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_2" >
      <attribute name="title" >
       <string>Keys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Mouseless control&lt;/span> &lt;/p>
&lt;table border="0" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px;" cellspacing="2" cellpadding="0">
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">right&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move map view to east&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">left&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move map view to west&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">up&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move map view to north&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">down&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move map view to south&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">+&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">zoom in&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">-&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">zoom out&lt;/p>&lt;/td>&lt;/tr>&lt;/table>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">3D View&lt;/span> &lt;/p>
&lt;table border="0" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px;" cellspacing="2" cellpadding="0">
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">m + left mouse&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move map&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">m + scroll mouse or page Up/Down&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">zoom map&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">home/end key&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">increase/decrease map area&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">l + left mouse&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move light source horizontaly&lt;/p>&lt;/td>&lt;/tr>
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">h + left mouse&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  &lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">move light source vertical&lt;/p>&lt;/td>&lt;/tr>&lt;/table>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignJustify|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IDlgConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IDlgConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
